apps/dash-3d-image-partitioning/assets/howto.md

```diff
@@ -0,0 +1,35 @@
+## What this app can do
+This dash app allows you to annotate automatically segmented brain regions. 
+You might, for example, want to annotate an area of hypoperfusion in an acutely infarcted brain and then download this annotation for further use. 
+This app allows you to:
+- interactively move through sliced representations of the brain
+- see an overlay of the automatically computed segmentation of the brain into superpixels (regions of the brain)
+- highlight one or several superpixels
+- see the highlighted superpixels rendered in a 3D glass brain representation
+- download a file with that contains the highlighted superpixels
+
+## How to use this app
+When you start the app, you will see two graphs that show axial (Top View) and saggital (Side View) slices through the same brain. 
+Overlayed on the brain in orange is an automatically computed segmentation of the brain into areas of similar intensity. 
+You can toggle this overlay on and off by clicking on the "Hide/Show Segmentation" button.
+These segmented areas are called superpixels, because they extend in all 3 dimensions of the brain volume. 
+
+You can move through the slices of the brain by dragging the sliders under each graph.
+
+To annotate an area of the brain (superpixel) you can mark the area by drawing a short line on it with your mouse cursor. 
+All areas that you draw on in this way will then be highlighted in the Top View and Side View representations.
+
+![Image](assets/howto-screenshot.png)
```

Comment:
On my app it doesn't show an image here, just the text "Image". Also I don't see a file `assets/howto-screenshot.png`, maybe it wasn't staged?

Reply:
Yeah I think I forgot to stage it. I made another commit with the image and I can see it on the playground deployment